starter/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.google.cloud.trace.adapters.zipkin</groupId>
+        <artifactId>parent</artifactId>
+        <version>0.2.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>starter</artifactId>
```

Comment:
ps it has been my experience that making ad-hoc servers ends up as a support burden, as there's rarely a reason to create a custom server.  starters are usually a means to create custom servers... are you sure you want to do this?

currently, we are using the "module" approach when creating custom servers, so it still ends up something that needs no custom build, rather composition at runtime.. like this: https://github.com/openzipkin/zipkin-aws/tree/master/autoconfigure/collector-sqs#running

If we want to bring the tomcat changes needed upstream to support that, seems great to me..